kubernetes/kubectl/tracing.yml

```diff
@@ -0,0 +1,338 @@
+---
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/path: /prometheus
+      prometheus.io/port: "9411"
+      fabric8.io/app-menu: management
+      fabric8.io/git-commit: 1f8f74e5427eae1ef9b9d743b133f0c971432591
+      fabric8.io/git-branch: release-v0.1.12
+      fabric8.io/scm-con-url: scm:git:git@github.com:fabric8io/kubeflix.git/zipkin
+      fabric8.io/scm-devcon-url: scm:git:git@github.com:fabric8io/kubeflix.git/zipkin
+      fabric8.io/scm-tag: 0.1.12
+      fabric8.io/scm-url: http://github.com/fabric8io/kubeflix/zipkin/
+      maven.fabric8.io/source-url: jar:file:/home/jenkins/workspace/ps_kubernetes-zipkin_PR-103-OSQCWHCK5H6C4WG3QXULIIRUVIJN6TOGHY6XMYURZ3YSKUYFNPHQ/zipkin/target/zipkin-0.1.12.jar!/META-INF/fabric8/kubernetes.yml
+    labels:
+      provider: fabric8
+      project: zipkin
+      version: 0.1.12
+      group: io.fabric8.zipkin
+      expose: "true"
+    name: zipkin
+  spec:
+    ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 9411
+      nodePort: 30335
+    selector:
+      project: zipkin
+      provider: fabric8
+      group: io.fabric8.zipkin
+    type: NodePort
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      fabric8.io/git-commit: 1f8f74e5427eae1ef9b9d743b133f0c971432591
+      fabric8.io/git-branch: release-v0.1.12
+      fabric8.io/scm-con-url: scm:git:git@github.com:fabric8io/kubeflix.git/zipkin-mysql
+      fabric8.io/scm-devcon-url: scm:git:git@github.com:fabric8io/kubeflix.git/zipkin-mysql
+      fabric8.io/scm-tag: 0.1.12
+      fabric8.io/scm-url: http://github.com/fabric8io/kubeflix/zipkin-mysql/
+      maven.fabric8.io/source-url: jar:file:/home/jenkins/workspace/ps_kubernetes-zipkin_PR-103-OSQCWHCK5H6C4WG3QXULIIRUVIJN6TOGHY6XMYURZ3YSKUYFNPHQ/mysql/target/zipkin-mysql-0.1.12.jar!/META-INF/fabric8/kubernetes.yml
+    labels:
+      provider: fabric8
+      project: zipkin-mysql
+      version: 0.1.12
+      group: io.fabric8.zipkin
+    name: zipkin-mysql
+  spec:
+    ports:
+    - port: 3306
+      protocol: TCP
+      targetPort: 3306
+    selector:
+      project: zipkin-mysql
+      provider: fabric8
+      group: io.fabric8.zipkin
+    type: LoadBalancer
+# below in the comment the original PersistentVolumeClaim but did not work
```

Comment:
@ffazzini why is this commented out? should we remove it instead?

Reply:
it was the original mysql-data config from fabric8 that gave the unmount issue, better to keep it there in case.